index.html

```diff
@@ -0,0 +1,124 @@
+        <img src="images/header-image.jpg" alt="Background image of brown dog.">
+        <h2 class="header-text">Seattle's premiere dog adoption agency</h2>
+      </article>
+
+      <article id="right-header">
+        <a href="checkout.html"><img src="images/cart-icon.svg" alt="Icon of a shopping cart.">$1234.56</a>
+        <p class="checkout"><a href="checkout.html">Checkout</a></p>
+      </article>
+    </header>
+
+<!--This is the navigation - current page has id of active for styling in navigation later-->
+    <ul>
+      <li class="navigation"><a id="active" href="index.html">Home /</a></li>
+      <li class="navigation"><a href="dogs.html">Dogs /</a></li>
+      <li class="navigation"><a href="blog.html">Blog</a></li>
+    </ul>
+
+<!-- This is the Adopt A Dog section -->
+    <section id="intro" class="grid-container-intro">
+      <img src="images/home-intro.jpg" alt="Picture of a boy hugging his dog.">
+      <h2>Adopt A Dog</h2>
+      <p>Dat tungg tho borkf many pats tungg adorable doggo very good spot long doggo, lotsa pats super chub long water shoob wrinkler. Pupperino clouds thicc heckin good boys such treat aqua doggo, lotsa pats boof adorable doggo very good spot. Waggy wags much ruin diet very good spot smol borking doggo with a long snoot for pats, adorable doggo borkf. Very taste wow the neighborhood pupper super chub you are doing me the shock borkdrive, wow such tempt stop it fren.</p>
+    </section>
+
+<!-- This is all the content for the grid of dogs - each element has a class for potential positioning later-->
+    <section id="dog-content" class="grid-container-dogs">
+      <article class="murphy">
+        <img class="murphy-img" src="images/murphy-card.jpg" alt="Image of Murphy who is a brown dog with white muzzle.">
+        <h3 class="murphy-name">Murphy</h3>
+        <p class="murphy-cost">Cost to Adopt: $123.45</p>
+        <p class="murphy-desc">Long doggo floofs very hand that feed shibe aqua doggo, noodle horse.</p>
+        <a class="murphy-adopt" href="index.html">Adopt</a>
+      </article>
+
+      <article class="poppy">
+        <img class="poppy-img" src="images/poppy-card.jpg" alt="Image of Poppy who is a white dog with brown ears.">
+        <h3 class="poppy-name">Poppy</h3>
+        <p class="poppy-cost">Cost to Adopt: $123.45</p>
+        <p class="poppy-desc">Long doggo floofs very hand that feed shibe aqua doggo, noodle horse.</p>
+        <a class="poppy-adopt" href="index.html">Adopt</a>
+      </article>
+
+      <article class="jack">
+        <img class="jack-img" src="images/jack-card.jpg" alt="Image of Jack who is a brown brown and white beagle.">
+        <h3 class="jack-name">Jack</h3>
+        <p class="jack-cost">Cost to Adopt: $123.45</p>
+        <p class="jack-desc">Long doggo floofs very hand that feed shibe aqua doggo, noodle horse.</p>
+        <a class="jack-adopt" href="index.html">Adopt</a>
```

Comment:
These don't link anywhere. They're just buttons. I'd swap to using a button element. Later on in the course it'll do something, but I'd steer you to using a button element.

Reply:
I am in alt! I didn't know about the button element and could not figure out what you were saying in the email you sent me. I have since googled this and see there is a button tag. I'll make these updates.